styles/_global.scss

```diff
@@ -0,0 +1,15 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: $font-primary;
+  height: 100%;
+  min-height: 100vh;
+  background-image: url('../static/assets/ui/QuizBackground.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
```

Comment:
Could anything happen if body takes 16px padding and has min-height of 100vh?

Reply:
I believe in our app the screen should take exactly 100vh and 100vw, scrolls should be considered bugs (at least on the bigger screens, responsive is another issue, but I would try to make it for 100vw and 100vh also).